### DISCUSSION TITLE
Could it make sense to give Passwordless Login Link Authentication arbitrary target URLs?

### DISCUSSION THREAD

**Discussion by manuelkiessling** [symfony/symfony#48630] [Q&A]
Unless I'm missing something obvious, the feature described in https://symfony.com/doc/current/security/login_link.html allows to create passwordless login links to a fixed target URL only, correct?

In my use case, I need to generate links which authenticate the user on arbitrary (project) URLs, by utilizing not a specific check route but instead allowing to add secure auth parameters to any route of my app.

I'd love to use the "official" feature, though. Would a PR be welcome which would allow to add something like a "post auth target url" on a per-login-link basis?

**Comment by lyrixx** [comment]
Yes you can! Good idea. 

But did you try to add a `targetPath` parameter to the URL?

**Reply by papppeter** [reply]
Hi @lyrixx ,

could you explain a little bit more? I would like to use it for SSO feature. need to have rediret info in url. is there a documentation to do i correclty?

thanks for your help

**Reply by papppeter** [reply]
i did find in code _target_path in defaultSuccessHandlerOptions. are you reffering to this? 

If you could add an example it would be great!

i would like to add redirect indicator param to the url, to be able to decide what happens after success.
